src/web/EmployerFavouritesWeb/Controllers/ApprenticeshipsController.cs

```diff
@@ -105,10 +106,122 @@ public async Task<IActionResult> TrainingProvider(string employerAccountId, stri
             var model = new TrainingProvidersViewModel
             {
                 EmployerAccountId = employerAccountId,
+                ApprenticeshipId = apprenticeshipId,
                 Items = response.Favourite.Providers.Select(mapper.Map).ToList()
             };
 
             return await Task.FromResult(View(model));
         }
+
+        [HttpGet("accounts/{employerAccountId:minlength(6)}/apprenticeships/{apprenticeshipId}/delete")]
+        public async Task<IActionResult> Delete(string employerAccountId, string apprenticeshipId)
+        {
+            if (!_paramValidator.IsValidEmployerAccountId(employerAccountId) ||
+                !_paramValidator.IsValidApprenticeshipId(apprenticeshipId))
+            {
+                _logger.LogDebug($"Invalid parameters in the following: {nameof(employerAccountId)}({employerAccountId}), {nameof(apprenticeshipId)}({apprenticeshipId})");
+                return BadRequest();
+            }
+
+            var response = await _mediator.Send(new ViewApprenticeshipFavouriteQuery
+            {
+                EmployerAccountId = employerAccountId,
+                ApprenticeshipId = apprenticeshipId
+            });
+
+
+            var mapper = new ApprenticeshipFavouriteMapper(_fatConfig);
+
+            var model = new TrainingProvidersViewModel
```

Comment:
The name of this model sound odd.

Reply:
it does a little, i just reused a existing model previously created by yourself :joy: